spec/features/users/user_signs_in_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+feature 'user signs in' do
+  scenario 'an existing user provides correct information' do
+    user = FactoryGirl.create(:user)
```

Comment:
Maybe pull this out into a `let` since you want a user for a couple of these tests.